SalesforceSDK/Salesforce.Sample.HybridMainPage.Universal/ApplicationInsights.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@kchitalia This project is only included in the VS2015 solution so I am not sure why we would not include it.  There is still a separate VS2013 solution.